infra/templates/azure-simple-hw/test/azure_simple_integration_test.go

```diff
@@ -0,0 +1,100 @@
+package test
```

Comment:
This looks like a fair bit of code that is unrelated to the assertions you want to make. Have you considered introducing a layer of abstraction that could be used to make the tests standardized and easy to write?

Reply:
that's a good point. As per our side chat this will be a top focus for our next sprint.